bed/bedPe/bedPe.go

```diff
@@ -0,0 +1,173 @@
+// Package bedpe provides functions for handling Bed Paired End format files, used for
+// handling pairs of genomic regions.
+
+package bedpe
```

Comment:
Can we also use "git mv" to change the "P" to lowercase in the directory name and file name?

Reply:
I think we need a couple commands to do this name change:
cd bed
mkdir bedpe
git mv bedPe/bedPe.go bedpe/bedpe.go